README.md

```diff
@@ -14,149 +14,3 @@ Note that the README in this repo is targeted at Calico docs contributors.
 
 **If you are looking for the repository formerly known as `projectcalico/calico`,
 it has been renamed to [`projectcalico/felix`](https://github.com/projectcalico/felix).**
-
```

Comment:
Also, I think this part should be updated:

```
 <blockquote>
 Note that the README in this repo is targeted at Calico docs contributors.
 <h1>Documentation for Calico users is here:<br><a href="http://docs.projectcalico.org">http://docs.projectcalico.org</a></h1>
 </blockquote>
```

To instead say something like:

```
See [contributing to the Calico docs](...) for information on how to make changes to the docs.projectcalico.org website.
```